Prep/Source/ImageTools/CropImages/source/CMakeLists.txt

```diff
@@ -4,3 +4,8 @@ project(CropImages)
 add_executable(CropImages CropImages.cxx)
 target_link_libraries(CropImages ${ITK_LIBRARIES} OptionParser)
 INSTALL(TARGETS CropImages  RUNTIME DESTINATION bin)
+
+add_executable(CropRawImages CropRawImages.cxx)
+target_link_libraries(CropRawImages ${ITK_LIBRARIES} OptionParser)
+INSTALL(TARGETS CropRawImages  RUNTIME DESTINATION bin)
```

Comment:
This tool is redundant. What is the difference between CropRawImages and CropImages. If the pixel type is the main different, you could use c++ templates. See the ReconstructMeanSurface tool for example.

Reply:
I have edited the CropImages to handle segmentation images, raw images or both and removed CropRawImages